main.go

```diff
@@ -54,6 +55,17 @@ func main() {
 	if err != nil {
 		log.Fatalf("failed to collect values: %v", err)
 	}
+
+	// calculate the version
+	version, err := versionFromFile(wd)
+	if err != nil {
+		log.Fatalf("failed to get version: %v", err)
+	}
+	if value, ok := collectGovvvDirective(args, flVersion); ok {
```

Comment:
I think you should directly check the flag first, then the file. If flag exists but the file doesn't, we'll return the error file not found error above. Also the .bats test isn't correctly covering this case.

Reply:
Having the flag take precedence over the file makes sense.  However, I don't think it should be an error if neither the flag and version file exist; that would be backward incompatible behavior.